docs/getting-started.md

```diff
@@ -1,18 +1,13 @@
 # Getting started
 
-## 1. Get your project added to the config preview
-As of July 2018 you will need to have your projects individually added to the configuration preview. Projects added will have their builds run through our new build ingestion and [configuration processing machinery](config-lifecycle.md). Talk to your account team at CircleCI. Once the system is stabilized we will either allow an opt-in to the new system or run all builds through config processing. 
-
-## 2. Get the new CLI 
+## 1. Get the new CLI 
 Follow the directions for [Getting Started](https://github.com/CircleCI-Public/circleci-cli/blob/master/README.md#getting-started) in the `circleci-cli` README.
 
-## 3. Set the "version" property to 2.1
+## 2. Set the "version" property to 2.1
```

Comment:
No need to make a change, but our general approach to numbering in markdown is make every bullet a `1.` and let the rendering take care of the ordering.